recipe/meta.yaml

```
@@ -1,36 +1,45 @@
 {% set name = "xmltodict" %}
-{% set version = "0.12.0" %}
-{% set sha256 = "50d8c638ed7ecb88d90561beedbf720c9b4e851a9fa6c47ebd64e99d166d8a21" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 341595a488e3e01a85a9d8911d8912fd922ede5fecc4dce437eb4b6c8d037e56
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: True  # [py<38]
 
 requirements:
   host:
     - pip
     - python
     - setuptools
+    - wheel
   run:
     - python
 
 test:
   imports:
     - xmltodict
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: https://github.com/martinblech/xmltodict
   license: MIT
-  license_file: LICENSE
+  license_family: MIT
+  license_url: https://github.com/martinblech/xmltodict/blob/v0.13.0/LICENSE
```

```suggestion
  license_file: LICENSE
```
As far as I can see, the license is included in the pypi archive.

---

oh okay! thanks